account-kit/smart-contracts/src/ma-v2/client/client.test.ts

```diff
@@ -1,77 +1,70 @@
 import {
-    custom,
-    keccak256,
-    parseEther,
-    publicActions,
-    toHex,
-    type Address,
-  } from "viem";
-  import { privateKeyToAccount } from "viem/accounts";
-  
-  import { LocalAccountSigner, type SmartAccountSigner } from "@aa-sdk/core";
-  
-  import {
-    createSMAV2SignerAccountClient,
-  } from "./client.js";
-  
-  import { local070InstanceOptSep } from "~test/instances.js";
-  import { setBalance } from "viem/actions";
-  import { accounts } from "~test/constants.js";
-  
-  describe("6900 RI installValidation Tests", async () => {
-    const instance = local070InstanceOptSep;
-    const client = instance.getClient().extend(publicActions);
+  custom,
+  keccak256,
+  parseEther,
+  publicActions,
+  toHex,
+  type Address,
+} from "viem";
+import { privateKeyToAccount } from "viem/accounts";
 
-    const signer: SmartAccountSigner = new LocalAccountSigner(
-      accounts.fundedAccountOwner
-    );
-  
-    it("should send a UO", async () => {
-      // Generate and add the secondary signer
-  
-      const provider = await givenConnectedProvider({ signer })
-  
-      console.log(signer);
-      console.log("provider: ", provider.getAddress());
+import { LocalAccountSigner, type SmartAccountSigner } from "@aa-sdk/core";
 
-      await setBalance(instance.getClient(), {
-        address: provider.getAddress(),
-        value: parseEther("2"),
-      });
-    
-      const target = "0x000000000000000000000000000000000000dEaD";
+import { createSMAV2AccountClient } from "./client.js";
 
-      const startBalance = await client.getBalance({ 
-        address: target,
-      })
+import { local070InstanceOptSep } from "~test/instances.js";
+import { setBalance } from "viem/actions";
+import { accounts } from "~test/constants.js";
 
-      const result2 = provider.sendUserOperation({
-        uo: {
+describe("6900 RI installValidation Tests", async () => {
+  const instance = local070InstanceOptSep;
+  const client = instance.getClient().extend(publicActions);
+
+  const signer: SmartAccountSigner = new LocalAccountSigner(
+    accounts.fundedAccountOwner
+  );
+
+  it("should send a UO", async () => {
+    // Generate and add the secondary signer
+
+    const provider = await givenConnectedProvider({ signer });
+
+    console.log(signer);
+    console.log("provider: ", provider.getAddress());
+
+    await setBalance(instance.getClient(), {
+      address: provider.getAddress(),
+      value: parseEther("2"),
+    });
+
+    const target = "0x000000000000000000000000000000000000dEaD";
+
+    const startBalance = await client.getBalance({
+      address: target,
+    });
+
+    const result2 = provider.sendUserOperation({
```

Comment:
⚠️ **[eslint]** <[@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'result2' is assigned a value but never used.
<!-- __reviewdog__:ChA0YTg0MDFmNmEyYTlmZDlhEgZyZGpzb24= -->